assets/questions/reverseWords.js

```diff
@@ -436,6 +448,52 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
       ]
     }],
     suiteLevelTests: [{
+      testSuiteIdsThatMustPass: [],
+      testSuiteIdsThatMustFail: ['SAMPLE_INPUT'],
+      messages: [
+        [
+          "Hmm, that code doesn't seem to work. Try taking a look at the ",
+          "sample inputs and outputs, then walk through your code to see what ",
+          "might be wrong."
+        ].join(''),
+        [
+          "Have you tried stepping through your code using the sample input ",
```

Comment:
Try stepping through your code using the sample input above. Note where and how it differs from the expected output.

Reply:
Hm -- for this one, the original text is from @rabidbit, and I prefer that version. I think, in the version you've proposed, it's not clear what "it" is.

I'm leaving this as-is for now. I'm generally happy to change it to anything, but this might be better as a discussion between you and @rabidbit.

Reply:
Sure, sounds good -- I think I generally am starting to prefer commands rather than rhetorical questions. I generally am leaning towards "clear and firm" rather than "interrogative", as it seems a bit more direct. The questions worry me that they might be interpreted as vaguely condescending.

Thoughts, @rabidbit? Not gonna let this block submit, though.

Reply:
I agree that questions can come across as condescending (or frustrating, if the user believes they have done what is being asked).  How about just the following:

"That code doesn't seem to work. Try taking a look at the sample inputs and outputs, and walk through your code to see what might be wrong."

(removes "Hmm," on first feedback and drops the inquiring feedback)

Reply:
Thanks for the resolution! Done.